spec/models/notification_spec.rb

```diff
@@ -25,6 +25,30 @@
     it { expect(subject).to allow_value("test@test.com").for(:email) }
     it { expect(subject).to_not allow_value("test@test").for(:email).with_message(/invalid/) }
 
-    it { expect(create(:sku_notification)).to validate_uniqueness_of(:email).scoped_to(:notifiable_id).with_message('notification has already been created.') }
+    it { expect(create(:sku_notification)).to validate_uniqueness_of(:email).scoped_to([:notifiable_id, :notifiable_type, :sent_at]).with_message('notification has already been created.') }
 
+    context 'skus and users can be associated to a notification' do
+      let(:duplicate_id) { 1000000009 }
+      let!(:sku)  { FactoryGirl.create(:sku, id: duplicate_id) }
+      let!(:user) { FactoryGirl.create(:user, id: duplicate_id) }
+
+      it 'should allow different models to be associated' do
+        sku_notification = create(:sku_notification, email: 'qwerty21@gmail.com', notifiable: sku)
+        expect(sku_notification.valid?).to be true
+        notification = build(:user_notification, email: 'qwerty21@gmail.com', notifiable: user)
```

Comment:
@drhenner  should really be using let method to create records to ensure the DatabaseCleaner wipes the db between specs.